test/models/CompanyProfile/CompanyProfile.test.ts

```diff
@@ -1,91 +1,148 @@
 import { CompanyProfile } from "../../../src/models/CompanyProfile";
 import { CompanyProfilePhoneNumber } from "../../../src/models/CompanyProfilePhoneNumber";
+import { CompanyProfilePhoto } from "../../../src/models/CompanyProfilePhoto";
 import Database from "../../../src/config/Database";
 
-beforeAll(async () => {
-  await Database.setConnection();
-});
 
-beforeEach(async () => {
-  await CompanyProfile.destroy({ truncate: true });
-});
+describe("CompanyProfile", () => {
+  beforeAll(async () => {
+    await Database.setConnection();
+  });
 
-afterAll(async () => {
-  await Database.close();
-});
+  beforeEach(async () => {
+    await CompanyProfile.destroy({ truncate: true });
+  });
 
-test("create a valid profile", async () => {
-  const companyProfile: CompanyProfile = new CompanyProfile({
+  afterAll(async () => {
+    await Database.close();
+  });
+
+  const companyProfileCompleteData = {
     cuit: "30711819017",
     companyName: "devartis",
     slogan: "We craft web applications for great businesses",
     description: "some description",
-    logo: "https://pbs.twimg.com/profile_images/1039514458282844161/apKQh1fu_400x400.jpg"
+    logo: "https://pbs.twimg.com/profile_images/1039514458282844161/apKQh1fu_400x400.jpg",
+    website: "https://www.devartis.com/",
+    email: "info@devartis.com"
+  };
+
+  it("create a valid profile", async () => {
+    const companyProfile: CompanyProfile = new CompanyProfile(companyProfileCompleteData);
+    const phoneNumber: CompanyProfilePhoneNumber = new CompanyProfilePhoneNumber({
+      phoneNumber: 43076555,
+      companyProfileId: 1
+    });
+    const photo: CompanyProfilePhoto = new CompanyProfilePhoto({
+      photo: `data:image/png;base64, iVBORw0KGgoAAAANSUhEUgAAAAUAAAAFCAYAAACNbyblAAAAHElEQVQI12P4//
+        8/w38GIAXDIBKE0DHxgljNBAAO9TXL0Y4OHwAAAABJRU5ErkJggg==`,
+      companyProfileId: 1
+    });
+    companyProfile.phoneNumbers = [ phoneNumber ];
+    companyProfile.photos = [ photo ];
+    await companyProfile.save();
+    expect(companyProfile).not.toBeNull();
+    expect(companyProfile).not.toBeUndefined();
+    expect(companyProfile.phoneNumbers).not.toBeUndefined();
+    expect(companyProfile.phoneNumbers).not.toBeNull();
+    expect(companyProfile.phoneNumbers).toHaveLength(1);
```

Comment:
aca hay varios expect redundantes, si pedis que tenga largo 1, el test ya va a fallar cuando es undefined o null

el ideal es 1 assert por test, tampoco digo llegar a eso, pero aca hay un wall of asserts

Reply:
Ademas esto es demasiado. Proba con algo como esto: https://github.com/fiuba-laboral-v2/back-end/pull/20/files#diff-b94053e4fc3b737f627063bf24b2f757R61